receiver/hostmetricsreceiver/e2e_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build e2e
+// +build e2e
```

Comment:
This won't be run on CI. We only call it for k8s components explicitly. Let's put `integration` tag here